test/stateful_dataloader/test_dataloader.py

```diff
@@ -1947,6 +1960,116 @@ def test_sampler_reproducibility(self):
                     ls[i].append(next(its[i]))
             self.assertEqual(ls[0], ls[1])
 
+    def test_initialization_StatefulDistributedSampler(self):
```

Comment:
Let's move all of these tests out to a new file called `test_sampler.py`. You can update https://github.com/pytorch/data/blob/main/.github/workflows/stateful_dataloader_ci.yml to call it in an additional step

Reply:
Created here: https://github.com/pytorch/data/blob/stateful_distributedsampler/test/stateful_dataloader/test_sampler.py

Added new line here: https://github.com/pytorch/data/blob/cdc5d318bab9432383e808dd25489677dd36edf9/.github/workflows/stateful_dataloader_ci.yml#L81